game-core/src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -90,19 +91,20 @@ public static void showMainFrame() {
     }
   }
 
-  private static void loadGame() {
+  private static void loadGameIfSpecified() {
     checkState(!SwingUtilities.isEventDispatchThread());
     gameSelectorModel.loadDefaultGameSameThread();
-    final String fileName = System.getProperty(TRIPLEA_GAME, "");
-    if (!fileName.isEmpty() && new File(fileName).exists()) {
+    final File saveGameFile = new File(System.getProperty(TRIPLEA_GAME, ""));
```

Comment:
Note: The `File` constructor doesn't throw an exception when using an empty path and seems to return false for `new File("").exists()`, so the empty check is redundant.

This way the cognitive complexity should go down.

Reply:
That's a good simplification :+1: